manifests/plugin/rabbitmq.pp

```diff
@@ -0,0 +1,74 @@
+# rabbitmq plugin
+# https://pypi.python.org/pypi/collectd-rabbitmq
+#
+# == Class collectd::plugin::rabbitmq
+#
+#  Configures rabbitmq metrics collection. Optionally installs the plugin
+#  Note, it is up to you to support package installation and sources
+#
+# === Parameters:
+# [*ensure*]
+#   String
+#   Passed to package and collectd::plugin resources ( both )
+#   Default: present
+#
+# [*interval*]
+#   Integer
+#   Interval setting for the plugin
+#   Default: undef
+#
+# [*manage_package*]
+#   Boolean
+#   Toggles installation of plugin. Please reference https://collectd-rabbitmq.readthedocs.org/en/latest/installation.html
+#   Default: undef
+#
+# [*package_provider*]
+#   String
+#   Passed to package resource
+#   Default: pip
+#
+# [*config*]
+#   Hash
+#   Contains key/value passed to the python module to configure the plugin
+#   Default: {
+#    'Username' => '"guest"',
+#    'Password' => '"guest_pass"',
+#    'Scheme'   => '"http"',
+#    'Port'     => '"15672"',
+#    'Host'     => "\"${::fqdn}\"",
+#    'Realm'    => '"RabbitMQ Management"',
+#   }
+#
+class collectd::plugin::rabbitmq (
+  $config           = {
+    'Username' => '"guest"',
+    'Password' => '"guest"',
+    'Scheme'   => '"http"',
+    'Port'     => '"15672"',
+    'Host'     => "\"${::fqdn}\"",
+    'Realm'    => '"RabbitMQ Management"',
+  },
+  $ensure           = 'present',
+  $interval         = undef,
+  $manage_package   = undef,
```

Comment:
Can you add a package_name parameter here and set it to collectd-rabbitmq by default?